engine/class_modules/sc_death_knight.cpp

```diff
@@ -5935,6 +6006,10 @@ struct obliterate_strike_t : public death_knight_melee_attack_t
     may_miss = false;
     weapon = w;
 
+    deaths_due_cleave_targets = as<int>(p -> spell.deaths_due -> effectN( 2 ).base_value()) + 
+                                  s_data -> effectN ( 1 ).chain_target() + 
+                                  as<int>(p -> find_spell( 49020 ) -> effectN ( 4 ).base_value());
```

Comment:
49020 is spec.obliterate, the data you're looking for is in 188290#4 (or spell.dnd_buff -> effectN( 4 ))

Reply:
I'm not really sure how I managed to do that one... but this is why we do review ;)